subsys/net/lib/downloader/src/transports/http.c

```diff
@@ -431,9 +435,7 @@ static int http_parse(struct downloader *dl, size_t len)
 	if (dl->progress + len != dl->file_size) {
 		if (http->ranged) {
 			http->ranged_progress += len;
-			if (http->ranged_progress < (dl->host_cfg.range_override
-							     ? dl->host_cfg.range_override
-							     : TLS_RANGE_MAX - 1)) {
```

Comment:
Leftover from earlier changes?

Reply:
`dl->host_cfg.range_override` will always be set when using TLS_RANGE_MAX.